sections/futures/TradingHistory/TradesHistoryTable.tsx

```diff
@@ -114,7 +114,7 @@ const TradesHistoryTable: FC<TradesHistoryTableProps> = ({ numberOfTrades, mobil
 						row.original.id !== NO_VALUE
 							? isL2Mainnet
 								? window.open(`${EXTERNAL_LINKS.Explorer.Optimism}/${row.original.id}`)
-								: window.open(`${EXTERNAL_LINKS.Explorer.OptimismKovan}/${row.original.id}`)
+								: window.open(`${EXTERNAL_LINKS.Explorer.OptimismGoerli}/${row.original.id}`)
```

Comment:
@Tburm here is the change.

Reply:
Looks like there might be some other issue with this logic, when I am connected to OP Goerli and click a trade it routes to `optimistic.etherscan`

Reply:
> Looks like there might be some other issue with this logic, when I am connected to OP Goerli and click a trade it routes to `optimistic.etherscan`

Thanks. I will investigate this issue.

Reply:
@Tburm When I switched to OP Goerli and check the spot history table. There is no trade at all. I also checked the subgraph endpoint but didn't get any results. https://api.thegraph.com/subgraphs/name/kwenta/optimism-goerli-main/
![image](https://user-images.githubusercontent.com/4819006/188836902-c1f52a5c-93d3-4a5d-9f94-9a90d37a7036.png)